test/unit/bxml/test_conference.py

```diff
@@ -47,20 +30,16 @@ def setUp(self):
         else:
-            expected = '<Conference mute="true" hold="false" callIdsToCoach="example-call-id" conferenceEventUrl="example.com/eventurl" conferenceEventMethod="POST" conferenceEventFallbackUrl="backupexample.com/eventurl" conferenceEventFallbackMethod="POST" username="user" password="pass" fallbackUsername="user" fallbackPassword="pass" tag="tag" callbackTimeout="5">conf1<PlayAudio>https://audio.url/audio1.wav</PlayAudio><StartRecording recordingAvailableUrl="example.com" recordingAvailableMethod="POST" username="user" password="pass" tag="tag" fileFormat="wav" multiChannel="true" /></Conference>'
+            expected = '<Conference mute="true" hold="false" callIdsToCoach="example-call-id" conferenceEventUrl="example.com/eventurl" conferenceEventMethod="POST" conferenceEventFallbackUrl="backupexample.com/eventurl" conferenceEventFallbackMethod="POST" username="user" password="pass" fallbackUsername="user" fallbackPassword="pass" tag="tag" callbackTimeout="5">conf1</Conference>'
         assert(expected == self.conference.to_bxml())
 
     def test_add_verb(self):
-        if os.environ['PYTHON_VERSION'] == '3.7':
-            expected = '<Conference callIdsToCoach="example-call-id" callbackTimeout="5" conferenceEventFallbackMethod="POST" conferenceEventFallbackUrl="backupexample.com/eventurl" conferenceEventMethod="POST" conferenceEventUrl="example.com/eventurl" fallbackPassword="pass" fallbackUsername="user" hold="false" mute="true" password="pass" tag="tag" username="user">conf1<PlayAudio>https://audio.url/audio1.wav</PlayAudio><StartRecording fileFormat="wav" multiChannel="true" password="pass" recordingAvailableMethod="POST" recordingAvailableUrl="example.com" tag="tag" username="user" /><SpeakSentence>Hello there.</SpeakSentence></Conference>'
-        else:
-            expected = '<Conference mute="true" hold="false" callIdsToCoach="example-call-id" conferenceEventUrl="example.com/eventurl" conferenceEventMethod="POST" conferenceEventFallbackUrl="backupexample.com/eventurl" conferenceEventFallbackMethod="POST" username="user" password="pass" fallbackUsername="user" fallbackPassword="pass" tag="tag" callbackTimeout="5">conf1<PlayAudio>https://audio.url/audio1.wav</PlayAudio><StartRecording recordingAvailableUrl="example.com" recordingAvailableMethod="POST" username="user" password="pass" tag="tag" fileFormat="wav" multiChannel="true" /><SpeakSentence>Hello there.</SpeakSentence></Conference>'
-        self.conference.add_verb(self.speak_sentence)
-        assert(expected == self.conference.to_bxml())
+        with pytest.raises(AttributeError):
```

Comment:
Your Test could follow same structure as your code in a way, that is, just like `add_verb` is only implemented in parent class `TerminalVerb` the test for it can be moved to a parent Test class `TerminalVerbBaseTestCase` removing Test duplication. Makes sense?

Reply:
Yes - great idea. Will reduce a lot of repeated tests